### PR TITLE
newer/maintained syntax highlighting

### PR DESCRIPTION
**Pull request by stvhay**
Migrates highlighter to highlight.js, which is maintained and supports more languages. Hope you like it.

### CODE REVIEW

**Comment by stvhay** [comment]
you can see what it looks like here https://bin.st5ve.com/

File: templates/view.html [hunk 1]
```diff
@@ -98,12 +97,7 @@
       <div class="pastey-title">
         <h1 class="mb-3 h2" id="pastey-title">{{ paste.title }}</h1>
       </div>
-      
-      <span style="font-weight:bold;">Format: </span>{{ paste.language }}</span>
-      {% if not paste.encrypted %}
```
Comment (Cesura) [inline templates/view.html:103]: Hi, thank you for all of the work recently. It seems you were making these changes against an older version of the code though, as this should not be removed. Please rebase your branch to make use of the latest version if possible.

Comment (stvhay) [reply templates/view.html:103]: I think I fixed it. The latest commit on this branch should be correct.

**Comment by stvhay** [comment]
Let me know if this addressed your comment. Happy to do the work. Kinda went on a coding bender. :)

**Comment by Cesura** [comment]
Yep, everything looks good now! Merging...

**Pull request merged**